invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -341,10 +341,24 @@
       "type": "array",
       "uniqueItems": true
     },
-    "_computed": {
+    "eitems": {
       "type": "object",
       "properties": {
-        "eitems": {
+        "total": { "type": "integer" },
+        "hits": {
```

Comment:
Do we consider only `hits` as a `ref` or shouldn't `total` also be a part of the resolved ref?

Same applies to `items`.

Reply:
yes, the entire `items` and `eitems` should be a $ref.